t/50mruby-http-request.t

```diff
@@ -197,29 +197,34 @@ EOT
                 for my $i (0..15) {
                     subtest "cl=$i" => sub {
                         my ($headers, $body) = run_prog("$curl_cmd $proto://127.0.0.1:$port/cl/$i");
-                        like $headers, qr{^HTTP/[^ ]+ 200\s.*\ncontent-length:\s*$i\r}is;
+                        like $headers, qr{^HTTP/[^ ]+ 200\b}ims;
+                        like $headers, qr{^content-length:\s*$i\b}ims;
                         is $body, substr "abcdefghijklmno", 0, $i;
                     }
                 };
                 for my $i (16..30) {
                     subtest "cl=$i" => sub {
                         my ($headers, $body) = run_prog("$curl_cmd $proto://127.0.0.1:$port/cl/$i");
-                        like $headers, qr{^HTTP/[^ ]+ 200\s.*\ncontent-length:\s*15\r}is;
+                        like $headers, qr{^HTTP/[^ ]+ 200\b}ims;
+                        like $headers, qr{^content-length:\s*15\b}ims;
                         is $body, "abcdefghijklmno";
                     }
                 };
             };
             subtest "chunked" => sub {
+                plan skip_all => "HTTP/3 is not yet supported" if $curl_cmd =~ /--http3/;
```

Comment:
We see bunch of these... could we do either or both of the following:
* use TODO rather than SKIP
* use one message throughout all the cases that we mark as skip or todo

The intent is to make sure they would be fixed in a follow-up PR.